cmd/feed-ingress/main.go

```diff
@@ -250,8 +259,13 @@ func main() {
 }
 
 func createIngressUpdaters() ([]controller.Updater, error) {
-	nginxConfig.IngressPort = ingressPort
+	nginxConfig.IngressPorts = append(nginxConfig.IngressPorts, nginx.IngressPortConf{Name: "http", Port: ingressPort})
```

Comment:
The append is unnecessary here. Just do `nginxConfig.IngressPorts := []IngressPortConf{{Name: "http", Port: port}}`.